OnlineVotingSystem/OnlineVotingSystem/Models/Complaints.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace OnlineVotingSystem.Models
+{
+    public class Complaints
+    {
+        [Key]
+        public int ComplaintID { get; set; }
```

Comment:
add summary for each property
ex: 
 /// <summary>
    /// The complaint id.
    /// </summary>